frontend/src/fetchers/ssdi.js

```diff
@@ -43,7 +42,7 @@ const allowedTopicsForQuery = {
     'domainInstructionalSupport',
     'createDate',
   ],
-  'qa-dashboard': [...QA_DASHBOARD_FILTER_CONFIG.map((filter) => filter.id),
```

Comment:
I believe changing this means you need to adjust the expected URL in a fetch mock, which is why your test is failing.

Reply:
thanks, I was truing to find where that was coming from